specifications/transactions/README.md

```diff
@@ -0,0 +1,61 @@
+# Diem Transactions and Payments
+
+## Overview
```

Comment:
I don't know exactly how you plan to structure this document but we should have a section at the top which probably indicates the late modified date (or there abouts) as well as a list of the DIPs (with links to them) that this Spec includes.